package.json

```diff
@@ -71,5 +66,5 @@
   "directories": {
     "test": "tests"
   },
-  "license": "BSD"
+  "license": "SEE LICENSE IN LICENSE"
```

Comment:
this is a valid SPDX expression to specify a file named LICENSE at the top level of the package